### PR TITLE
Update CosmosDB Annotatiosn and delete NotificationHubOutput

### LINKED ISSUE #53: Delete NotificationHubOutput
Not supported in V2 yet

### PR DESCRIPTION
**Pull request by pragnagopa**
Fixes #53 #48

### CODE REVIEW

**Comment by pragnagopa** [comment]
@brettsam  - Can you please review?

File: src/main/java/com/microsoft/azure/functions/annotation/CosmosDBOutput.java [hunk 1]
```diff
@@ -12,66 +12,116 @@
...
+     * @return Throughput of the created collection.
+     */
+    int collectionThroughput();
+
+    /**
+     * Enable to use with Multi Master accounts.
+     * 
+     * @return whether to Multi Master accounts
+     */
+    boolean useMultipleWriteLocations() default false;
```
Comment (pragnagopa) [inline src/main/java/com/microsoft/azure/functions/annotation/CosmosDBOutput.java]: Should this be added to Trigger and Input binding?

**Comment by pragnagopa** [comment]
@jdneo , @Flanker32  FYI

File: src/main/java/com/microsoft/azure/functions/annotation/CosmosDBOutput.java (hunk 1 above)
Comment (brettsam) [reply src/main/java/com/microsoft/azure/functions/annotation/CosmosDBOutput.java]: Looping in @ealsur to validate, but I don't believe so. This is only for writing documents.

File: src/main/java/com/microsoft/azure/functions/annotation/CosmosDBTrigger.java [hunk 2]
```diff
@@ -38,57 +44,176 @@
...
+    boolean startFromBeginning() default false;
+
+    /**
+     * Defines preferred locations (regions) for geo-replicated database accounts in
+     * the Azure Cosmos DB service. Values should be comma-separated. example,
+     * PreferredLocations = "East US,South Central US,North Europe"
+     * 
+     * @return preferred locations (regions) for geo-replicated
+     */
+    String PreferredLocations();
```
Comment (brettsam) [inline src/main/java/com/microsoft/azure/functions/annotation/CosmosDBTrigger.java]: Should this be lower-cased?

**Comment by pragnagopa** [comment]
Thanks @brettsam. Merging this PR. @ealsur - let me know if you any other feedback.

**Pull request merged**